src/modules/tgBot/scenes/linkAccounts.scene.ts

```diff
@@ -30,15 +34,22 @@ export class LinkAccountsScene {
     @Sender('first_name') firstName: string,
     @Sender('last_name') lastName: string,
     @Sender('username') userName: string,
+    @Sender('phone_number') phoneNumber: string,
     @Sender('id') userId: number
   ): Promise<string> {
-    let substrateAccountId = null;
     let processingMessage = null;
-    substrateAccountId = ctx.state.command.args[0];
+    const linkingTmpIdOrAddress = ctx.state.command.args[0];
 
-    if (!substrateAccountId) {
+    if (!linkingTmpIdOrAddress) {
       await ctx.reply(
-        '⚠️ The account ID has not been provided along with the command.'
+        `Let's rock! Go to your Profile settings in grill.chat application, find "Connect Telegram" button and copy 
```

Comment:
Maybe extract messages in one separate folder, will be easy for editing in the future (and for Beef review also will be cool)